sdk/dotnet/GetInstanceType.cs

```diff
@@ -61,20 +61,20 @@ public GetInstanceTypeArgs()
     [OutputType]
     public sealed class GetInstanceTypeResult
     {
-        public readonly Outputs.GetInstanceTypeAddonsResult Addons;
+        public readonly ImmutableArray<Outputs.GetInstanceTypeAddonResult> Addons;
```

Comment:
There's a bunch of these changes - they seem breaking. DO you know why we get them?